atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/qos/ThriftObjectSizeUtils.java

```diff
@@ -53,6 +54,28 @@ public static long getApproximateSizeOfMutationMap(Map<ByteBuffer, Map<String, L
         return approxBytesForKeys + approxBytesForValues;
     }
 
+    public static Map<String,Long> computeSizePerTable(Map<ByteBuffer, Map<String, List<Mutation>>> mutation_map) {
+        Map<String, Long> tableToSize = new HashMap<>();
+
+        mutation_map.forEach((key, tableToMutations) -> {
+            long keySize = ThriftObjectSizeUtils.getByteBufferSize(key);
+
+            tableToMutations.forEach((table, mutations) -> {
+                if (!tableToSize.containsKey(table)) {
+                    tableToSize.put(table, 0L);
+                }
+
+                Long size = tableToSize.get(table);
```

Comment:
could use `getOrDefault(..., 0L)` and drop the above loop

Reply:
Done.